layouts/authors/list.html

```diff
@@ -0,0 +1,72 @@
+{{ define "name" }}
+{{ .Title }} &ndash; {{ .Site.Title }}
+{{ end }}
+
+
+{{ define "header" }}
+<div class="sm:pt-5 
+
+relative bg-[#1D65A6]">
+    {{ partial "site-nav.html" . }}
+</div>
+{{ end }}
+
+{{ define "main" }}
+
+<div class="max-w-7xl mx-auto px-4 my-auto relative">
+    <div class="flex basis-3 items-start py-4 md:p-16 flex-col md:flex-row">
+        {{ if .Params.name }}
+        <div class="w-full md:w-1/2">
+            <div class="">
+                {{ if .Params.profile }}
+                <img src="{{ .Params.profile | absURL}}" alt="" class='rounded-t-xl covatar w-full object-cover' />
+                {{ else }}
+                <img src="/img/butterfly.png" alt=""  />
```

Comment:
I've added this photo as more of a placeholder for contributing authors, if we don't have a photo for them. We can replace this with something else.

Reply:
That works for me!